.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: cpp
-dist: bionic
+dist: focal
```

Comment:
`<filesystem` isn't available in `gcc-7+`, just upgrade ubuntu to focal dist which have `gcc-8+`